WinUIGallery/SearchResultsPage.xaml.cs

```diff
@@ -116,8 +114,8 @@ private void BuildFilterList(string queryText)
                 {
                     // Display informational text when there are no search results.
                     VisualStateManager.GoToState(this, "NoResultsFound", false);
-                    var textbox = NavigationRootPage.GetForElement(this)?.PageHeader?.GetDescendantsOfType<AutoSuggestBox>().FirstOrDefault();
-                    textbox?.Focus(FocusState.Programmatic);
+                    //var textbox = NavigationRootPage.GetForElement(this)?.PageHeader?.GetDescendantsOfType<AutoSuggestBox>().FirstOrDefault();
+                    //textbox?.Focus(FocusState.Programmatic);
```

Comment:
Intentional?

Reply:
![image](https://user-images.githubusercontent.com/9866362/216822183-5ec113f9-0843-4c61-a051-77967b19cd8c.png)

Seems to be dead code.. for whenever this was still a thing.